common-crafting.lic

```diff
@@ -12,26 +12,26 @@ module DRCC
     ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
   end
 
-  def find_empty_crucible
+  def find_empty_crucible(idle_room = nil, crucibles)
```

Comment:
Would it be any easier if you passed in a hometown string as an argument and had these helpers do the lookup?